src/test/rgw/test_rgw_string.cc

```diff
@@ -12,6 +12,7 @@
  *
  */
 
+#include <errno.h>
```

Comment:
thanks @wjwithagen. i had that `#include "rgw/rgw_string.h"` first to make sure the header could be included on its own. it sounds like it's the header that has a dependency on `<errno.h>`, so i'd prefer to move the include there